GA of Keep on latest version (#5214)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/2024-release-notes.md

## May 2024

- <expandable alt_header="Update: Keep on latest version" > 

>   The new **Keep on latest version** setting is now GA, no longer in Public Preview. It allows dbt Labs to handle version upgrades for you.

Reviewer: I'd prefer one quick sentence:
```suggestion
The **Keep on latest version** setting is now Generally Available (previously Public Preview).
```
Then let the paragraph below do all the explanation
